Mayflower Release 9.27.0

avrilpearl opened the pull request:
## 9.27.0 (10/08/2019)

### Changed

- (Patternlab) [CampaignVideo]: Updated background color variables to match color variable names in MF. Added top/bottom padding to the component. #776

### Fixed

- (React) [EmergencyHeader, DecorativeLink] Fix decorative link wrapping issue in alert header and download link. #775
- (React, Patternlab) [Circleci] Fix auto generate changelog for dependabot prs (followup #722) #765

### Security

- (React) [Dependency] Bump sshpk from 1.13.1 to 1.16.1 in /react #592

rbayliss commented:
I discovered two minor issues while reviewing this release:

1.  How to page icons show up in black/white on Firefox:
<img width="1319" alt="screen shot 2018-04-18 at 3 49 25 pm" src="https://user-images.githubusercontent.com/654407/38955180-d66e1a20-4321-11e8-8c16-524a50cafe98.png">

2. After filtering a location listing, the header has extra space at the top.  
<img width="1213" alt="screen shot 2018-04-18 at 4 02 28 pm" src="https://user-images.githubusercontent.com/654407/38955215-f7944a1c-4321-11e8-961d-7658e3ab4841.png">

I don't feel either of these are release blockers, but I wanted to note them in case anyone feels differently.

rbayliss commented:
Also, Melissa discovered an issue with the Org page's display of news items in IE11:
![screen shot 2018-04-18 at 4 32 00 pm](https://user-images.githubusercontent.com/654407/38956648-3a92f21a-4326-11e8-960d-28f291f1a77e.png)

clairesunstudio committed:
Merge pull request #774 from massgov/master

Sync Develop with Master Release 9.26.0

avrilpearl committed:
Patternlab/DP-16102: Update campaign video styling (#776)

* Update campaign video component

* Update campaign landing page example

* Add pr #

* Update screenshots

clairesunstudio committed:
fix decorative display in alert header (#775)

* fix decorative display in alert header

* remove line-flex override

* revert change

* approve screenshots

* add changelog

dependabot-preview[bot] committed:
[Security] Bump sshpk from 1.13.1 to 1.16.1 in /react (#592)

* [Security] Bump sshpk from 1.13.1 to 1.16.1 in /react

Bumps [sshpk](https://github.com/joyent/node-sshpk) from 1.13.1 to 1.16.1. **This update includes security fixes.**
- [Release notes](https://github.com/joyent/node-sshpk/releases)
- [Commits](https://github.com/joyent/node-sshpk/compare/v1.13.1...v1.16.1)

* Add changelog

Commit:
Add changelog to changelogs/dependabot.md

avrilpearl committed:
consolidate changelogs

mrossi113 reviewed (approved):
👍 LGTM!
